notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -907,6 +901,17 @@ public void removeDraftOfNote(Page page, String username) throws WikiException {
    */
   @Override
   public void removeDraftOfNote(Page page) throws WikiException {
+    List<DraftPage> draftPages = dataStorage.findDraftPagesByTargetPage(page);
```

Comment:
```suggestion
    List<DraftPage> draftPages = dataStorage.getDraftsOfPage(Long.parseLong(page.getId()));
```
IMO you can use the existing method dataStorage.getDraftsOfPage instead of adding a new one